cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -497,6 +508,22 @@ open class Node @Inject constructor(private val project: Project) {
         }
     }
 
+    internal fun installPostgresDriver() {
```

Comment:
I don't understand why PostgreSQL is "special". What about Oracle? And MySQL? Does _everyone_ get built-in support like this?

Supporting PostgreSQL is fine and presumably desirable. But I think the plugin should be allowing people to "Put a JDBC driver of their own choosing in the correct place". We _do not_ want to need to release a new version of the Gradle plugins just to upgrade people's versions of PostgreSQL.